src/api/interactivity.js

```diff
@@ -75,7 +75,9 @@ export default class Interactivity {
     * Moreover, the order of the features in the events will be determined by the order of the layers in this list.
     *
     * @param {carto.Layer|carto.Layer[]} layerList - {@link carto.Layer} or array of {@link carto.Layer}, events will be fired based on the features of these layers. The array cannot be empty, and all the layers must be attached to the same map.
-    *
+    * @param {object} [options={}] - Object containing interactivity options
+    * @param {boolean} [options.defaultCursor=false] - A boolean flag indicating if the cursor should change when the mouse is over a feature.
```

Comment:
I would rename the parameter and improve its documentation. It is not clear what is the meaning of true and what is the meaning of false. I would name something like `autoChangePointer`.

Reply:
I agree ☝️